pyeudiw/satosa/backend.py

```diff
@@ -33,8 +33,10 @@
 )
 from pyeudiw.storage.db_engine import DBEngine
 from pyeudiw.storage.exceptions import StorageWriteError
+from pyeudiw.federation.schemas.wallet_relying_party import WalletRelyingParty
 
 from pydantic import ValidationError
+from pprint import pformat
```

Comment:
if pformat uses multiplelines, It cannot be used for logs because it may run in conflicts with the syslog approached and the common logging systems

then: do not USE multiple .warnings, please use format strings in a single warning

Reply:
Resolved in [4165826](https://github.com/italia/eudi-wallet-it-python/pull/138/commits/4165826cec281b07f5f4e0912984b240e1170ebd) and [364331e](https://github.com/italia/eudi-wallet-it-python/pull/138/commits/364331ed8cf82e0594b6369383d977c670a0f9a2)